.flake8

```diff
@@ -31,12 +31,26 @@ per-file-ignores =
     __init__.py: F401
     test/**: F821
     test/**/__init__.py: F401,F821
-    torch/**: TOR901
     torch/utils/cpp_extension.py: B950
     torchgen/api/types/__init__.py: F401,F403
     torchgen/executorch/api/types/__init__.py: F401,F403
     test/dynamo/test_higher_order_ops.py: B950
     torch/testing/_internal/dynamo_test_failures.py: B950
+    # TOR901 is only for test, we want to ignore it for everything else.
+    # It's not easy to configure this without affecting other per-file-ignores,
+    # so we explicitly list every file where it's violated outside of test.
+    torch/__init__.py: F401,TOR901
+    torch/_custom_op/impl.py: TOR901
+    torch/_export/serde/upgrade.py: TOR901
+    torch/_functorch/vmap.py: TOR901
+    torch/_inductor/test_operators.py: TOR901
+    torch/_library/abstract_impl.py: TOR901
+    torch/_meta_registrations.py: TOR901
+    torch/_prims/__init__.py: F401,TOR901
+    torch/_prims/rng_prims.py: TOR901
+    torch/ao/quantization/fx/_decomposed.py: TOR901
+    torch/distributed/_functional_collectives.py: TOR901
+    torch/distributed/_spmd/data_parallel.py: TOR901
```

Comment:
I feel like we want a per-file-include option. It's totally fine for torch.library to be used in torch/* and we don't want to confuse people who begin using it in a new torch/* file (because it seems common)

Reply:
`per-file-include` would be nice, but that is not present in flake8. TorchFix is just a flake8 plugin here, we can't add options.